src/components/TabTraining.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div class="tab training">
     <h1>Training Options</h1>
+    <h2>Argument Parser</h2>
```

Comment:
Should we call this section as configuration system ?
Options could be: 
- argparse + yaml
- python-fire + yaml
and when hydra is available:
- hydra


What do you think ?

Reply:
@vfdev-5 I think Heading should be `Configuration System` but options seem to be fine as yaml is a default package in python so it does not need to be mentioned here, also it will be a bit easier with `meta.json` logic